package.json

```diff
@@ -421,7 +437,9 @@
     },
     "dependencies": {
         "@hubspot/cli-lib": "^4.1.6-beta.0",
+        "axios": "^1.2.2",
```

Comment:
Since this is just for the serverless function should this be externalized in the webpack build?

Reply:
Since it's used in `src/lib/panels/feedback.ts` it needs to be a dep for the repo.

Reply:
ah, sorry I missed that! Out of curiosity, any reason not to use `fetch` on the VS Code side?

Reply:
Sorry for the "unresolve comment" just wanted to follow up to see if we could avoid adding this dep? We already have `node-fetch` which is used for snippet generation and vscode makes `fetch` available as well.

Reply:
I had tried to use `node-fetch` before, but I'm running into an issue with using it. I think it may be because we are working within the panel, which is a webview, we aren't technically in node anymore.

Reply:
It was an import issue. I was able to remove `axios` in favor of `node-fetch`.